src/stream/src/executor/aggregation/mod.rs

```diff
@@ -352,8 +353,8 @@ pub fn generate_agg_schema(
 
 /// Infer column desc for state table.
 /// The column desc layout is
-/// [ `group_key` (only for hash agg) / `sort_key` (only for simple agg) / `value`(the agg call
-/// return type)].
+/// [ `group_key` (only for hash agg) / `sort_key` (only for max/mix without append only) /
```

Comment:
```suggestion
/// [ `group_key` (only for hash agg) / `sort_key` (only for extreme) /
```

Reply:
It is a fix when implementing #2999 . No direct case relates to it.